README.md

```diff
@@ -120,12 +127,14 @@ make test
 Start chain 1 (terminal 1)
 ```shell
 make setup-sol-cli
-make start-eth
+cd ../chainbridge-solidity
+make start-ganache
 ```
 
 Start chain 2 (terminal 2)
 ```shell
-PORT=8546 make start-eth
+cd ../chainbridge-solidity
+PORT=8546 make start-ganache
```

Comment:
Please remove the whole `Simulations` section. They are being worked on elsewhere